src/main/java/com/unzer/payment/Cancel.java

```diff
@@ -38,7 +38,7 @@ public String getUrl() {
 
         return partialResult
                 .replaceAll(TRANSACTION_TYPE_TOKEN, getTransactionType() == null ? TransactionType.AUTHORIZE : getTransactionType())
-                .replaceAll(TRANSACTION_ID_TOKEN, getId() == null ? "" : getId());
+                .replaceAll(TRANSACTION_ID_TOKEN + "/", getId() == null ? "" : getId());
```

Comment:
There seem to have been kind of a bug. It created an url with // if no transaction id was present. But now i realize that i need to adjust it because it seem to be broken now in case the id is present.

Reply:
its now adjusted to "keep" the slash when an id is present.